debian/not-installed

```diff
@@ -1,53 +1,185 @@
+lib/systemd/system/mariadb-extra.socket # Installed by rules file
+lib/systemd/system/mariadb-extra@.socket # Installed by rules file
 lib/systemd/system/mariadb.service # Installed by rules file
 lib/systemd/system/mariadb.socket # Installed by rules file
-lib/systemd/system/mariadb-extra.socket # Installed by rules file
 lib/systemd/system/mariadb@.service # Installed by rules file
 lib/systemd/system/mariadb@.socket # Installed by rules file
-lib/systemd/system/mariadb-extra@.socket # Installed by rules file
-usr/bin/mysql_config # We already have the MariaDB variant
-usr/bin/mytop # Mytop is distributed from a separate source package
+usr/bin/mariadb-embedded # Shipping the embedded server in distro packaging does not make sense
+usr/bin/mysql_config # Debian packaging has mysql_config as symlink to mariadb_config
+usr/bin/mysql_embedded # Symlink to mariadb-embeded which is intentionally not included
 usr/bin/sst_dump # Use the one from rocksdb-tools package
-usr/lib/mysql/plugin/JavaWrappers.jar # These are only built if JNI/libjawt.so is installed from e.g. openjdk-8-jre-headless
-usr/lib/mysql/plugin/JdbcInterface.jar # These are only built if JNI/libjawt.so is installed from e.g. openjdk-8-jre-headless
-usr/lib/sysusers.d/sysusers.conf
-usr/lib/tmpfiles.d/tmpfiles.conf
 usr/lib/*/libdbbc.a # ColumnStore header file
 usr/lib/*/libidbboot.a # ColumnStore header file
 usr/lib/*/libprocessor.a # ColumnStore header file
 usr/lib/*/libwe_xml.a # ColumnStore header file
-usr/share/doc/mariadb-server-10.9/COPYING
-usr/share/doc/mariadb-server-10.9/COPYING.AGPLv3
-usr/share/doc/mariadb-server-10.9/COPYING.GPLv2
-usr/share/doc/mariadb-server-10.9/COPYING.thirdparty
+usr/bin/test-connect-t
+usr/bin/wsrep_sst_backup
+usr/lib/mysql/plugin/type_test.so
+usr/lib/sysusers.d/mariadb.conf # Not used (yet) in Debian systemd
+usr/lib/tmpfiles.d/mariadb.conf # Not used (yet) in Debian systemd
+usr/sbin/rcmysql
+usr/share/doc/mariadb-server-10.9/COPYING (related file: "debian/tmp/usr/share/mysql/mroonga/COPYING")
 usr/share/doc/mariadb-server-10.9/CREDITS
-usr/share/doc/mariadb-server-10.9/EXCEPTIONS-CLIENT
 usr/share/doc/mariadb-server-10.9/INSTALL-BINARY
-usr/share/doc/mariadb-server-10.9/PATENTS
 usr/share/doc/mariadb-server-10.9/README-wsrep
-usr/share/groonga/COPYING
-usr/share/groonga-normalizer-mysql/lgpl-2.0.txt
+usr/share/doc/mariadb-server-10.9/README.md
+usr/share/doc/mariadb-server-10.9/THIRDPARTY
 usr/share/groonga-normalizer-mysql/README.md
+usr/share/groonga-normalizer-mysql/lgpl-2.0.txt
+usr/share/groonga/COPYING
 usr/share/groonga/README.md
+usr/share/man/man1/mariadb-embedded.1 # Shipping the embedded server in distro packaging does not make sense
+usr/share/man/man1/mysql_embedded.1 # Symlink to mariadb-embeded.1 which is intentionally not included
 usr/share/man/man1/my_safe_process.1
+usr/share/man/man1/mysql-stress-test.pl.1
 usr/share/man/man1/mysql.server.1
+usr/share/man/man3/mariadb_cancel.3
```

Comment:
shouldn't these end up in libmariadb-dev package?

Reply:
Yes, all files in the not-install file that have no comment probably should be either added to some package or a comment to the line added explaining why we intentionally don't ship it in the Debian packaging.

Since this was already merged, I suggest somebody from the Foundation review the remaining `not-installed` file contents and move the lines to appropriate `*.install` file (or add a comment why not).